balancer/pickfirst/pickfirstleaf/pickfirstleaf.go

```diff
@@ -61,6 +62,14 @@ var (
 // TODO: change to pick-first when this becomes the default pick_first policy.
 const logPrefix = "[pick-first-leaf-lb %p] "
 
+type ipAddrType int
+
+const (
+	ipTypeUnknown ipAddrType = iota
+	ipv4
+	ipv6
```

Comment:
Maybe we should be consistent with the enum names. The unknown enum has the `ipType` prefix while the other two don't. If we go with `ipAddrFamily` as the name for the type, maybe the individual enum values could be `ipAddrFamilyUnknown`, `ipAddrFamilyV4` and `ipAddrFamilyV6`. If we decide to go with `ipAddrType`, we could still take a similar approach.